test/easyconfigs/easyconfigs.py

```diff
@@ -1764,6 +1764,21 @@ def template_easyconfig_test(self, spec):
             fail_msg += f"'{orig_val}' vs '{dumped_val}'"
             failing_checks.append(fail_msg)
 
+    # don't allow updating of $PYTHONPATH with standard lib/python*/site-packages path,
+    # since that's already taken care of by EasyBuild framework now
+    # (cfr. https://github.com/easybuilders/easybuild-framework/pull/4539)
+    modextrapaths = ec.get('modextrapaths', {})
+    regex = re.compile(r'^lib*/python[0-9]\.[0-9]+/site-packages$')
+    for key, value in modextrapaths.items():
+        value = resolve_template(value, ec.template_values)
```

Comment:
`ec.get` has a `resolve` parameter defaulting to `True`, so it seems the value is already resolved at this point, isn't it?

If it was indeed required I'd suggest putting it below the check to avoid resolving every value without using almost any

Reply:
Ah, maybe, I didn't actually verify that tbh... Will check